source/Engine.cpp

```diff
@@ -508,6 +509,17 @@ void Engine::Step(bool isActive)
 	wasActive = isActive;
 	Audio::Update(center);
 
+	if(GameData::IsLoaded())
+	{
+		Track::GameState state = Track::GameState::IDLE;
+		if(hadHostiles)
```

Comment:
Just wondering what does hadHostile mean?
Because it seems like it already existed before, so idk if it matches what we want with it (like mb hostiles being in system is enough for hostile track?)

Reply:
This is what it means, the name is confusing though

Reply:
A comment then x)

Reply:
done